cmd/client-s3.go

```diff
@@ -2313,3 +2315,55 @@ func (c *S3Client) SetVersioning(ctx context.Context, status string) *probe.Erro
 	}
 	return probe.NewError(err)
 }
+
+// GetEncryption - gets bucket encryption info.
+func (c *S3Client) GetEncryption(ctx context.Context) (algorithm, keyID string, err *probe.Error) {
+	bucket, _ := c.url2BucketAndObject()
+	if bucket == "" {
+		return "", "", probe.NewError(BucketNameEmpty{})
+	}
+
+	config, e := c.api.GetBucketEncryption(ctx, bucket)
+	if e != nil {
+		return "", "", probe.NewError(e)
+	}
+	for _, rule := range config.Rules {
+		algorithm = rule.Apply.SSEAlgorithm
+		if rule.Apply.KmsMasterKeyID != "" {
+			keyID = rule.Apply.KmsMasterKeyID
+			break
```

Comment:
I do not understand this exactly, are we supposed to have only one rule for now ?

Reply:
yes